src/test/java/bio/terra/pact/provider/SnapshotsApiControllerTest.java

```diff
@@ -73,7 +75,17 @@ public static SelectorBuilder consumerVersionSelectors() {
     // runs, and your consumer is publishing such Pacts under their feature branch name, you can add
     // the following to the SelectorBuilder:
     //   .branch("consumer-feature-branch-name")
-    return new SelectorBuilder().mainBranch().deployedOrReleased();
+    // New comments.
```

Comment:
This comment should probably be merged with the one above it, or outright overwrite it.

Reply:
Consolidated and revised the comment. @jladieu feel free to copy the latest comment to your TODO on `workspacemanager` repo.